.gitlab/ci/helper.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+######################################################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC and other CARE developers.
+# See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+######################################################################################
+
+mkdir build
```

Comment:
If possible, I would change this to mkdir build-cuda or mkdir build-blueos-nvcc10.1.243_clang8.0.1.cmake or something more descriptive.

Reply:
I changed it to build-cuda. When we actually have more than one build, I will definitely be more specific.

Reply:
Thanks!